docs/03/b.md

```diff
@@ -66,5 +66,28 @@ project with a `.g8` extension) you can test it with the actual g8
 runtime. When you're ready, add your template project to the
 [the wiki][wiki] so other giter8 users can find it.
 
+### Using Mill
+
+There is also an external [Mill plugin][mill-plugin] that can be used to test
+your templates as well. An example setup can be found below:
+
+```scala
+import \$ivy.`io.chris-kipp::mill-giter8::0.2.0`
```

Comment:
is this `\` intentionally here?

Reply:
Yea,  if you look further up all the $ are escaped. I _believe_ because it uses either itself or the same mechanism to copy over files so $ needs to be escaped. Although I now that I look I don't see this on the website. @eed3si9n is there anything special needed to get the website to be updated?

Reply:
This repo used to be set up to publish on its own, but something regressed it seems. I just published it manually - http://www.foundweekends.org/giter8/testing.html#Using+Mill